Dockerfile

```diff
@@ -1,66 +1,97 @@
+FROM ubuntu:18.04 AS builder
+
+LABEL maintainer="team@appwrite.io"
+
+ARG TESTING=false
+
+ENV TZ=Asia/Tel_Aviv \
+    DEBIAN_FRONTEND=noninteractive \
+    PHP_VERSION=7.3 \
+    PHP_REDIS_VERSION=3.1.2
+
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends --no-install-suggests ca-certificates software-properties-common wget curl git openssl && \
```

Comment:
`ca-certificates` is needed so you can handle https protocol because we are dealing with https links used by apt and wget.
for `git` I cannot remember the reason behind adding it but I assume that composer might need it if it's going to install packages through git itself